src/ontology/mondo-edit.obo

```diff
@@ -263186,18 +263186,19 @@ property_value: confidence "3.533333333333334" xsd:double
 
 [Term]
 id: MONDO:0013209
-name: non-alcoholic fatty liver disease
-def: "A term referring to fatty replacement of the hepatic parenchyma which is not related to alcohol use." [NCIT:C84444]
+name: metabolic dysfunction-associated steatotic liver disease
+def: "Metabolic dysfunction-associated steatotic liver disease (MASLD, formerly known as nonalcoholic fatty liver disease or NAFLD) is a type of liver disease that is not caused by alcohol. It typically does not cause symptoms in the early stages, but it can cause health problems due to fat accumulation, inflammation, and scarring in the liver." [https://www.verywellhealth.com/how-dangerous-is-non-alcoholic-fatty-liver-disease-2223390]
```

Comment:
We usually don't start a definition by repeating the disease name. But in this case, since we include the "formerly known" names, I think it is ok.

Reply:
I thought the same thing